.storybook/components/Docs/Guidelines/Theming.stories.mdx

```diff
@@ -70,6 +70,36 @@ Once run, you will have a set of theme files written to the configured `dest` pa
 
 To use, add this file to your core app root file **after** where the imported EDS's `@chanzuckerberg/eds/index.css` file is inserted.
 
+### eds-import-from-Figma
+
+Command to run: `npx eds-import-from-figma`
+
+Designers can define and tweak the values of a custom theme. For this reason, we want to automate the application of those changes to the project's existing theme file.
+
+To use this command, there are a few prerequisites:
+
+* In Figma, install the [Export/Import Variables Plugin][export-import-plugin].
+
+With the prerequisites set up, you can download a JSON file, containing the existing token definitions and values. Here's how:
+
+* Open the design(s) in Figma
+* Open the Resources Panel in the toolbar (or use <kbd>Shift + I</kbd>)
+* Activate the "Export/Import Variables" plugin by first clicking the "Plugins" tab, then the "Export/Import Variables" option.
+* Once it opens, click the "Export..." button for the Listed collections and save to your local machine
+* Finally, us the above command, by passing in the file path to each collection.
```

Comment:
For testing purposes, would it be value for point solutions to have something like this in CI?

CI Check `ensure theme matches design?`
1. run `npx eds-import-from-figma`
2. check diff of generated file to existing file in version control

Reply:
In order to run in CI, we would need to have the theme file being fetched from figma, or stored somewhere in the consuming project. The former depends on the type of licensing with Figma (e.g., some do not allow access to the REST API, which can let you pull down such a figma JSON file). The latter I'm not yet sure about... 

What would be useful is running `eds-apply-theme` in husky, which would be easy enough to forget to do, OR running that after a successful import. It will take the imported values and generate the outputs. 

Let's chat more about this to see if we are aligned. I do have some followup tasks around integration of this tool, but wanted to get the core of it in place for discussion and testing